scripts/scr_array_functions/scr_array_functions.gml

```diff
@@ -80,7 +80,10 @@ function array_to_string_list(_array) {
         return;
     }
     for (var i = 0; i < array_length(_array); i++) {
-        _string_list += _array[i];
+		if (_array[i] == 0) { // top of a stack trace from debug_get_callstack
+			continue;
+		}
+        _string_list += string(_array[i]);
```

Comment:
Are you sure that you want to do this here? I'm concerned that if you run this function outside `debug_get_callstack`, expecting a 0 in the list as well, this will cause it to skip it.
Maybe do this in the error throw function instead, pre-bake the array before this function.

Reply:
moved in [0cbca70](https://github.com/EttyKitty/ChapterMaster/pull/68/commits/0cbca7097e52c19650666e726c4d6b108d802e1b)